pkg/controllers/csi/metadata/migrations.go

```diff
@@ -0,0 +1,93 @@
+package metadata
+
+import (
+	"path/filepath"
+
+	dtcsi "github.com/Dynatrace/dynatrace-operator/pkg/controllers/csi"
+	"gorm.io/gorm"
+)
+
+func dataMigration(tx *gorm.DB) error {
+	var dynakubes []Dynakube
+
+	tx.Table("dynakubes").Find(&dynakubes)
+
+	for _, d := range dynakubes {
+		tc := TenantConfig{
+			Name:                        d.Name,
+			TenantUUID:                  d.TenantUUID,
+			ConfigDirPath:               filepath.Join(filepath.Join(dtcsi.DataPath, d.TenantUUID), d.Name, dtcsi.SharedAgentConfigDir),
+			DownloadedCodeModuleVersion: d.LatestVersion,
```

Comment:
this could be `ImageDigest` as well

[Relevant code](https://github.com/Dynatrace/dynatrace-operator/blob/efe236974140ed208f4dfda94ad0351599068770/pkg/controllers/csi/provisioner/controller.go#L225-L228)

Context: Downloaded images and dowloaded zips used to be stored in a different location, and the way we differentiated "where to look" is by checking if the `ImageDigest` or the `LatestVersion` is set.
But as of right now we only store things in the same folder. (also we are possible moving away from digests, so that is that :D) 
- This "where to look" was/is only relevant during download, mount, and cleanup, and not during unmount, so that is why you do not see it in the `volumes` table. (its confusing, I know 😅 )

Reply:
fixed in 2639aa76